### PR TITLE
feat: p1 checkout and user profile page

### PR DESCRIPTION
**Pull request by bhtibrewal**
https://user-images.githubusercontent.com/42600164/155199602-53a6fe4a-86c5-4077-9dbe-aae9a38aa108.mp4

### CODE REVIEW

File: pages/checkout/checkout.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+          <div class="total-part">
+            <span><strong>Total</strong></span>
+            <span><strong>₹817,000</strong></span>
+          </div>
+          <hr />
+          <label>
+            <input type="checkbox" />
+            <span class="checkbox-text">This artwork is a gift. </span>
+          </label>
+
```
Comment (gauravsinhaweb) [inline pages/checkout/checkout.html]: remove spacing for consistency.

File: pages/checkout/checkout.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    <!--********** main ************-->
+
+    <main class="main user-page">
+      <section class="sec user-header-sec flex-col">
+        <div class="avatar-text avatar-m">BT</div>
+        <h1>Hello Bhavika</h1>
+        <p class="body-l">Artsy collector since 22/2/2022</p>
+      </section>
+      <h1>Secure Checkout in 2 steps</h1>
+
```
Comment (gauravsinhaweb) [inline pages/checkout/checkout.html:73]: remove spacing for consistency.

File: pages/cart/cart.html [hunk 3]
```diff
@@ -46,67 +61,66 @@
...
-                  Visit ten places on our planet that are undergoing the biggest
-                  changes today.
-                </div>
-                <div class="price-sec">
-                  <h3>$18</h3>
-                  <span class="strike-price">$22</span>
-                </div>
+      <section class="cart-sec">
+        <p class="body-l">My Cart(2)</p>
+        <div class="card w-40 hori">
```
Comment (pradipgharge) [inline pages/cart/cart.html:68]: can name it hz instead of hori

File: pages/cart/cart.html [hunk 4]
```diff
@@ -46,67 +61,66 @@
...
+          <div class="content">
+            <img
+              class="card-img"
+              src="https://render.fineartamerica.com/images/images-new-artwork/images-medium-5/la-porta-rossa-sulla-salita-guido-borelli.jpg?v=2"
+              alt=""
```
Comment (pradipgharge) [inline pages/cart/cart.html:73]: put some name in alt tag

File: pages/user-profile/user-profile.html [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+    <title>Artsy-shop</title>
+  </head>
+
+  <body class="body">
+    <!--********** Header ************-->
+    <header class="arstyShop-header header">
+      <a href="/">
+        <h3><i class="fa-brands fa-shopify"></i>Artsy Shop</h3>
+      </a>
+
```
Comment (gauravsinhaweb) [inline pages/user-profile/user-profile.html:23]: remove spacing for consistency.

File: pages/user-profile/user-profile.html [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+        <h1>Hello Bhavika</h1>
+        <p class="body-l">Artsy collector since 22/2/2022</p>
+      </section>
+      <h1>My Account</h1>
+
+      <!-- grid  -->
+      <section class="user-grid">
+          <!-- contact details from -->
+        <form class="form sec contact-dets">
+
```
Comment (gauravsinhaweb) [inline pages/user-profile/user-profile.html:75]: remove spacing for consistency.

**Review by gauravsinhaweb — APPROVED** [review]

**Review by pradipgharge — APPROVED** [review]
Looks good, left a few comments. Do check

**Comment by gauravsinhaweb** [comment]
One suggestion in the UI, in the `https://deploy-preview-6--artsy-shop.netlify.app/pages/checkout/checkout.html` give `Secure Checkout in 2 steps` a h2 class instead of h1 because you already have a title.

**Pull request merged**